bin/public_interface.py

```diff
@@ -212,13 +215,19 @@ def main() -> None:
     subparsers = parser.add_subparsers(dest="command")
     extract = subparsers.add_parser("extract", help="Extract public interfaces")
     extract.add_argument("--module", help="The module to extract public interfaces", type=str, default="samtranslator")
+    extract.add_argument(
+        "--skipped-modules",
+        help="The modules separated by ',' that should be skipped",
+        type=str,
+        default="samtranslator.internal",
+    )
     check = subparsers.add_parser("check", help="Check public interface changes")
     check.add_argument("original_json", help="The original public interface JSON file", type=Path)
     check.add_argument("new_json", help="The new public interface JSON file", type=Path)
     args = parser.parse_args()
 
     if args.command == "extract":
-        scanner = InterfaceScanner()
+        scanner = InterfaceScanner(skipped_modules=args.skipped_modules.split(","))
```

Comment:
Should we trim the resulting strings in case the customer adds spaces between the comma and the next item?

Reply:
no longer apply due to nargs